<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ssmClass</class>
 <widget class="QMainWindow" name="ssmClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Steam StorageManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="fastStorageLabel">
      <property name="text">
       <string>Fast Storage: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="massStorageLabel">
      <property name="text">
       <string>Mass Storage: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="moveToFastButton">
        <property name="text">
         <string>&lt;--</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveToMassButton">
        <property name="text">
         <string>--&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QListView" name="fastStorageView">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QListView" name="massStorageView">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ssm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fastStorageView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>massStorageView</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>massStorageView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>fastStorageView</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>903</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
